tests/CI/run_docker_setup.sh

```diff
@@ -188,12 +188,11 @@ function prepareEnvironment() {
   docker cp "$SERVERCONFIG" server:"$WORKSPACE/CONFIG"
   docker cp "$CLIENTCONFIG" client:"$WORKSPACE/CONFIG"
 
-  # if the DIRACOS_TARBALL_PATH is a local directory, copy it over
-  if [ -d "${DIRACOS_TARBALL_PATH}" ]; then
+  # Copy DIRACOS_TARBALL_PATH if it is a local directory containing a DIRACOS tarball
+  if ls "${DIRACOS_TARBALL_PATH}"/diracos-*.tar.gz 1> /dev/null 2>&1; then
```

Comment:
is the md5 file mandatory or not ? If yes, we may as well check it here too ?

Reply:
It technically is so far as I know but I don't see the point in checking it here. It will fail soon after with an obvious error message when `dirac-install` is ran.

It would be more confusing if it ignores your custom DIRACOS build because you forgot the MD5 file (or if `dirac-install` is modified to support a different checksum and the check here wasn't updated to match).

Reply:
Fine with me